### ISSUE TITLE
How to read and convert the image to match the one on database to mark the attendance 

### ISSUE THREAD

**Issue by Hojeah2663** [Spidy20/Attendace_management_system#4]
Now I have this specific error at which the image from the cam can’t be converted to gray image so it brings this error 

File "AMS_Run.py", line 347, in Fillattendances
    gray = cv2.cvtColor(im, cv2.COLOR_BGR2GRAY)
cv2.error: OpenCV(4.2.0) C:\projects\opencv-python\opencv\modules\imgproc\src\color.cpp:182: error: (-215:Assertion failed) !_src.empty() in function 'cv::cvtColor'
 Please help

**Comment by Spidy20** [comment]
It's not getting frame, that's the error.

**Comment by Hojeah2663** [comment]
Yes so how do we fix it ? Because the camera is open and the video is captured 

    On Tuesday, June 9, 2020, 02:36:55 PM GMT+8, Kushal Bhavsar <notifications@github.com> wrote:  
 
 

It's not getting frame, that's the error.

**Comment by Hojeah2663** [comment]
How do we fix it?

**Comment by Spidy20** [comment]
I can't say about it, because i haven't seen your code and error. Send me
some SC.
